msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -7,49 +7,79 @@ ms.localizationpriority: high
+> The behavior of deep links is dependent on a number of factors. The following list outlines the behavior of deep links on Teams entities.
 >
 > **Tab**:  
-> ✔ Directly navigates to the deeplink url.
+> ✔ Directly navigates to the deep link url.
 >
 > **Bot**:  
-> ✔ Deeplink in card body: Opens in browser first.  
-> ✔ Deeplink added to OpenURL action in Adaptive Card: Directly navigates to the deeplink url.  
+> ✔ Deep link in card body: Opens in browser first.  
+> ✔ Deep link added to OpenURL action in Adaptive Card: Directly navigates to the deep link url.  
 > ✔ Hyperlink markdown text in the card: Opens in browser first.  
 >
 > **Chat**:  
-> ✔ Text message hyperlink markdown: Directly navigates to deeplink url.  
-> ✔ Link pasted in general chat conversation: Directly navigates to deeplink url.
+> ✔ Text message hyperlink markdown: Directly navigates to deep link url.  
+> ✔ Link pasted in general chat conversation: Directly navigates to deep link url.
+>
+> For Teams apps extended across Microsoft 365 (Outlook/Office) behavior is dependent upon where the deep link is pointing relative to where the Teams app is running. If the Teams app is running within the host the deep link is pointing to it will open directly within the host. If the Teams app is running in a host that differs from where the link is pointing, it will open first in the browser.
 
-## Deep linking to your tab
+## Deep linking with the Teams JavaScript client SDK
 
-You can create deep links to entities in Teams. This is used to create links that navigate to content and information within your tab. For example, if your tab contains a task list, team members can create and share links to individual tasks. When you select the link, it navigates to your tab that focuses on the specific item. To implement this, you add a **copy link** action to each item, in whatever way best suits your UI. When the user takes this action, you call `shareDeepLink()` to display a dialog box containing a link that the user can copy to the clipboard. When you make this call, you also pass an ID for your item, which you get back in the [context](~/tabs/how-to/access-teams-context.md) when the link is followed and your tab is reloaded.
+The Microsoft Teams JavaScript client SDK (TeamsJS) simplifies the process of creating and using deep links to entities in Teams and Teams apps. Deep links can be used for scenarios such as navigating to content and information within your tab or even launching a chat dialog. While it is possible, and for some scenarios necessary, to manually create a deep link, the SDK provides strongly typed APIs that make for an improved experience. These APIs are particularly useful for Teams apps that might be run in other hosts (Outlook, Office), as they provide a way to check that the capability being used is supported by that host. The following sections highlight the uses of deep links and how the SDK facilitates their usage.
```

Comment:
Couple of points here:

1) In back-compat scenarios ("old Teams"), we still make deep links inside of the strongly typed capabilities (like appInstallDialog). In all other hosts including "new Teams" we're no longer creating deep links inside of the API. This might be a good example of not needing to expose this amount of detail anymore. I think we could say something like "For any scenarios that you used to craft deep links for (such as launching a chat dialog), you can now use strongly typed capabilities instead". I don't think developers need to know that secretly sometimes that means that deep links are being made for them.
2) I might say "These APIs are required for Teams apps that might be run in other hosts" instead of "These APIs are particularly useful"
3) Might end with something like "The following sections highlight how scenarios that used deep links in 1.x have changed in 2.x and how the SDK facilitates their usage" assuming that matches the intention of the article

Reply:
Thanks for the feedback. I moved a lot of this content to the beginning of the topic and reworded things some. For number 2, I said recommended (vs required) since I believe deep links do work. If that's incorrect, just let me know.